core/trino-main/src/main/java/io/trino/server/ProtocolConfig.java

```diff
@@ -49,7 +49,7 @@ public int getPreparedStatementCompressionThreshold()
     }
 
     @Config("protocol.v1.prepared-statement-compression.length-threshold")
-    @ConfigDescription("Prepared statements longer than the configured value will be compressed")
+    @ConfigDescription("Prepared statements longer than the configured value are compressed")
```

Comment:
"Compression is applied to prepared statements longer than the configured value."

[This also better matches the description in line 66.]